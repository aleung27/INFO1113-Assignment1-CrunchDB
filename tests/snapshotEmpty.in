SET HELLO 3
SNAPSHOT
DEL HELLO
SNAPSHOT
SNAPSHOT
DROP 1
SNAPSHOT
SNAPSHOT
LIST SNAPSHOTS
