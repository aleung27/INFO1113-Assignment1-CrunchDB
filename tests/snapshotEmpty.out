> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> saved as snapshot 3

> ok

> saved as snapshot 4

> saved as snapshot 5

> 5
4
3
2

